lib/RoadizCoreBundle/src/Routing/NodeRouteHelper.php

```diff
@@ -39,11 +41,12 @@ public function __construct(
     public function getController(): ?string
     {
         if (null === $this->controller) {
-            if (!$this->node->getNodeType()->isReachable()) {
+            $nodeType = $this->nodeTypesBag->get($this->node->getNodeTypeName());
+            if (!$nodeType->isReachable()) {
                 return null;
             }
             $controllerClassName = $this->getControllerNamespace().'\\'.
-                StringHandler::classify($this->node->getNodeType()->getName()).
+                StringHandler::classify($nodeType).
```

Comment:
Nope.
`$nodeType` is not a string. It's a `NodeType` your phpstan should have warned you here.